README.md

```diff
@@ -0,0 +1,149 @@
+   cd bank-of-anthos/
+   ```
+
+3. Set the Google Cloud project and region and ensure the Google Kubernetes Engine API is enabled.
+
+   ```sh
+   export PROJECT_ID=<PROJECT_ID>
+   export REGION=us-central1
+   gcloud services enable container.googleapis.com \
+     --project=${PROJECT_ID}
+   ```
+
+   Substitute `<PROJECT_ID>` with the ID of your Google Cloud project.
+
+4. Create a GKE cluster and get the credentials for it.
+
+   ```sh
+   gcloud container clusters create-auto bank-of-anthos \
+     --project=${PROJECT_ID} --region=${REGION}
+   ```
+
+   Creating the cluster may take a few minutes.
+
+5. Deploy Bank of Anthos to the cluster.
+
+   ```sh
+   kubectl apply -f ./extras/jwt/jwt-secret.yaml
+   kubectl apply -f ./kubernetes-manifests
+   ```
+
+6. Wait for the pods to be ready.
+
+   ```sh
+   kubectl get pods
+   ```
+
+   After a few minutes, you should see the Pods in a `Running` state:
+
+   ```
+   NAME                                  READY   STATUS    RESTARTS   AGE
+   accounts-db-6f589464bc-6r7b7          1/1     Running   0          99s
+   balancereader-797bf6d7c5-8xvp6        1/1     Running   0          99s
+   contacts-769c4fb556-25pg2             1/1     Running   0          98s
+   frontend-7c96b54f6b-zkdbz             1/1     Running   0          98s
+   ledger-db-5b78474d4f-p6xcb            1/1     Running   0          98s
+   ledgerwriter-84bf44b95d-65mqf         1/1     Running   0          97s
+   loadgenerator-559667b6ff-4zsvb        1/1     Running   0          97s
+   transactionhistory-5569754896-z94cn   1/1     Running   0          97s
+   userservice-78dc876bff-pdhtl          1/1     Running   0          96s
+   ```
+
+7. Access the web frontend in a browser using the frontend's external IP.
+
+   ```sh
+   kubectl get service frontend | awk '{print $4}'
+   ```
+
+   Visit `http://EXTERNAL_IP` in a web browser to access your instance of Bank of Anthos.
+
+8. Once you are done with it, delete the GKE cluster.
+
+   ```sh
+   gcloud container clusters delete bank-of-anthos \
+     --project=${PROJECT_ID} --region=${REGION}
+   ```
+
+   Deleting the cluster may take a few minutes.
+
+## Additional deployment options
+
+- **Workload Identity**: [See these instructions.](/docs/workload-identity.md)
+- **Cloud SQL**: [See these instructions](/extras/cloudsql) to replace the in-cluster databases with hosted Google Cloud SQL.
+- **Multi Cluster with Cloud SQL**: [See these instructions](/extras/cloudsql-multicluster) to replicate the app across two regions using GKE, Multi Cluster Ingress, and Google Cloud SQL.
+- **Istio**: [See these instructions](/extras/istio) to configure an IngressGateway.
+- **Anthos Service Mesh**: ASM requires Workload Identity to be enabled in your GKE cluster. [See the workload identity instructions](/docs/workload-identity.md) to configure and deploy the app. Then, apply `extras/istio-manifests/` to your cluster to configure frontend ingress.
```

Comment:
nit: Since both of them at same time make less sense and IngressGateway is now a part of the service KRM, it might make sense to combine them under ASM or to make a more clear separation.

Reply:
Perhaps, but that's beyond the scope of this PR (this section in particular has not changed -- the diff is just messing up since the README moved around elsewhere)